src/pivot-table/data/extract-headers.ts

```diff
@@ -0,0 +1,25 @@
+import { NxDimensionInfo } from "../../types/QIX";
+import { Cell } from "../../types/types";
+import createCell from "./cell";
+
+const extractHeaders = (qDim: NxDimensionInfo[], rowCount: number, colCount: number): Cell[][] => {
+  const ary: Cell[][] = [];
+
+  for (let colIdx = 0; colIdx < colCount; colIdx += 1) {
+    for (let rowIdx = 0; rowIdx < rowCount; rowIdx += 1) {
```

Comment:
`++` is nice here

Reply:
Again it's a lint rule that prevents me from doing that :(